packages/arb-token-bridge-ui/tests/e2e/specs/readClassicDeposits.cy.ts

```diff
@@ -43,19 +43,14 @@ describe('Read classic deposit messages', () => {
     cy.restoreAppState()
   })
 
-  afterEach(() => {
-    cy.saveAppState()
-  })
-
   context('User has classic ETH deposit transaction', () => {
     // log in to metamask
     before(() => {
-      cy.login({ networkType: 'L1' })
-    })
-
-    after(() => {
-      // after all assertions are executed, logout and reset the account
-      cy.logout()
+      cy.login({
```

Comment:
We can move this logic from both the tests to `beforeEach` and we can remove `restoreAppState` since these tests need to be run in isolation.

Reply:
These are not the same - the first `cy.login` includes `shouldChangeNetwork: true`.

This is because we now share metamask state between tests. The issue is explained [here](https://github.com/OffchainLabs/arbitrum-token-bridge/pull/582/files#diff-a1b664142558b990753488c3fd65511e12bf2a8b10f9e900c5fb10d43a253d98R26). Hopefully synpress upgrade fixes this so that we can run `cy.getNetwork()` check without having to pass a boolean for it.

Reply:
Okay, if they're different then makes sense to keep it within their respective tests.